k8s/cmd/nsmd-k8s/main.go

```diff
@@ -73,18 +71,11 @@ func main() {
 	}()
 
 	span.Logger().Infof("Start prefix service")
-	prefixService, err := prefixcollector.NewPrefixService(config)
+	err = prefixcollector.NewExcludePrefixServer(config)
```

Comment:
Question: Why did you choose to put this in nsmd-k8s rather than nsmd ?

Reply:
AFAIK, nsmd-k8s is for k8s-specific stuff and getting prefixes for pod subnet and service subnet are k8s-specific.

Reply:
prefixService is moved to dedicated pod